_data/simple-icons.json

```
@@ -6359,6 +6359,17 @@
             "hex": "25313C",
             "source": "https://www.iris.co.uk"
         },
+        {
+            "title": "ISC2",
+            "hex": "468145",
+            "source": "https://www.isc2.org",
+            "aliases": {
+                "aka": [
+                    "(ISC)²",
+                    "ISC Squared"
+                ]
```

```suggestion
                "aka": [
                    "(ISC)²"
                ]
```

Refer to https://github.com/simple-icons/simple-icons/pull/9134#discussion_r1290958345